docs/Reference/MPS/configuration.md

```diff
@@ -14,6 +14,7 @@
 | MPS_WEB_PORT               | 3000                 | |
 | MPS_WEB_ADMIN_USER         | n/a                  | Specifies the username for API authentication |
 | MPS_WEB_ADMIN_PASSWORD     | n/a                  | Specifies the password for API authentication |
+| MPS_WEB_AUTH_ENABLED       | n/a                  | Specifies whether or not to enable MPS authentication |
```

Comment:
Here's a way to structure the comments: 
| MPS_WEB_ADMIN_USER         | n/a                  | Username for API authentication |
| MPS_WEB_ADMIN_PASSWORD     | n/a                  | Password for API authentication |
| MPS_WEB_AUTH_ENABLED       | n/a                  | MPS authentication: enable(true) or disable(false) |


Here's another way. Like, with verbs and user direction:
| MPS_WEB_ADMIN_USER         | n/a                  | Add your username for API authentication |
| MPS_WEB_ADMIN_PASSWORD     | n/a                  | Add a password for API authentication |
| MPS_WEB_AUTH_ENABLED       | n/a                  | Enable(true) or disable(false) the MPS authentication |

I'm just guessing on the enable/disable. 

The reason I'm showing two ways is that I don't know the tone and flavor of these files. What's traditional for open source, yadah, yadah. 

I mean, I want the whole comment structure to be revamped to be consistent and precise, but in the interest of time...